tools/cli/bin/jetpack.js

```diff
@@ -29,34 +57,21 @@ try {
 /**
  * Standardizes the cwd for the process. Allows `jetpack` cli to run correctly from any location in the repo.
  */
-
 process.chdir( fileURLToPath( new URL( '../../..', import.meta.url ) ) );
 
+/**
+ * Checks to make sure we're on the right version of various tools.
+ */
 try {
-	const cliRouter = await import( '../cliRouter.js' );
-	cliRouter.cli();
-} catch ( error ) {
-	const identity = v => v;
-	const chalk = await import( 'chalk' ).then(
-		m => m.default,
-		() => ( {} )
-	);
-
-	if ( error.code === 'ERR_MODULE_NOT_FOUND' ) {
-		// if pnpm install hasn't been run, the import() will fail here.
-		console.error(
-			'Something is missing from your install. Please run `pnpm install` and try again.'
-		);
-
-		// Print the original error's message too, as sometimes the error isn't fixed by `pnpm install`.
-		console.error(
-			( chalk.grey || identity )( `The original error message was ${ error.message }` )
-		);
-	} else {
-		console.error( error );
-		console.error(
-			( chalk.bold || identity )( 'Something unexpected happened. See error above.' )
-		);
+	if ( ! ( await compareToolVersions() ) && ! process.env.JETPACK_CLI_IGNORE_VERSION_CHECKS ) {
+		console.error( '(Ignore the above errors by setting JETPACK_CLI_IGNORE_VERSION_CHECKS=1)' );
```

Comment:
I don't know if we should show the errors anyway if the user has decided to ignore them? But that's more of a question than a request for change.

Reply:
Personally I'd rather they be there to remind the user about this even if they don't want to fix them right now. But maybe "ignore" is a poor word.